blog/_src/posts/2016-11-16-constructive-galois-connections.md

```diff
@@ -0,0 +1,167 @@
+    Title: Understanding Constructive Galois Connections
+    Date: 2016-11-16T00:00:00
+    Tags: icfp, galois connection, adjunction, category thoery, math
+
+One of my favorite papers at ICFP 2016 (in lovely
+[Nara, Japan](http://conf.researchr.org/home/icfp-2016)) was
+[Constructive Galois Connections ...](http://dl.acm.org/citation.cfm?id=2951934)
```

Comment:
Yeah, let's avoid paywall-looking links on the blog when we can. Thanks for catching it.